import-service/services/catalog-batch-process/catalogBatchProcess.api.ts

```diff
@@ -0,0 +1,61 @@
+import superagent from 'superagent';
+import { SQSHandler } from 'aws-lambda';
+import { SNS } from 'aws-sdk';
+
+export const catalogBatchProcess: SQSHandler = async (
+  event,
+  context,
+  callback
+) => {
+  console.log('🚀 ~ file: catalogBatchProcess.api.ts ~ line 9 ~ event', event);
+  try {
+    const sns = new SNS();
+    await Promise.all(
+      event.Records.map(async (record) => {
+        console.log(
+          '🚀 ~ file: catalogBatchProcess.api.ts ~ line 13 ~ record',
+          record
+        );
+        const body = JSON.parse(record.body);
+        console.log(
+          '🚀 ~ file: catalogBatchProcess.api.ts ~ line 18 ~ body',
+          body
+        );
+        await superagent
```

Comment:
I think, you create an overhead here by invoking one Lambda from another. Probably, it  would be faster, easier and chipper (after reducing number of invocations) to move product creation code to service and use it directly from both handlers

Reply:
Yes, thanks! Fix this, move it to product-service